airbyte-ci/connectors/pipelines/README.md

```diff
@@ -11,44 +11,17 @@ This documentation should be helpful for both local and CI use of the CLI. We in
 ## How to install
 ### Requirements
 * A running Docker engine with version >= 20.10.23
-* Python >= 3.10
-* [pipx](https://pypa.github.io/pipx/installation/)
 
 
 ## Install or Update
```

Comment:
Can we reference the `clean` and `check` commands here?

Reply:
I mention them in troubleshooting.
https://github.com/airbytehq/airbyte/blob/a4713ef2725ddc1d92d678b1c04a738fb4c1eb01/airbyte-ci/connectors/pipelines/README.md#troubleshooting

But can give them a more standout area